Gemfile

```diff
@@ -6,10 +6,8 @@ require_relative 'config/boot_settings'
 source 'https://rubygems.org'
 
 case SETTINGS[:rails]
-when '5.1'
```

Comment:
this also needs adjustements elsewhere

Reply:
Specifically in `config/settings.yaml.example`